api/controllers/auth/login.js

```diff
@@ -46,12 +48,14 @@ module.exports = function (req, res) {
          if (err) {
             req.ab.log("error logging in:", err);
             res.ab.error(err, 401);
+            authLogger(req, "Local auth FAILED");
             return;
          }
          req.ab.log("successful auth/login");
          req.session.tenant_id = req.ab.tenantID;
          req.session.user_id = user.uuid;
          res.ab.success({ user });
+         authLogger(req, "Local auth successful");
```

Comment:
since the goal is to trigger `fail2ban`, is there any benefit for logging successful auths?  or are we just using up more disk space?

Reply:
With fail2ban, successful auth entries are ignored. So you are right, there is no benefit there. 

However, my thinking was that successful entries could be used to reset the fail count of an IP address. For example, if I got my password wrong 4 times, but got it right on the 5th, a reset would save me from being banned if I somehow logged out immediately and messed it up again the next time.

Fail2ban doesn't support that. But maybe it will in the future? Or we may find a competing solution that does have that feature.